runhouse/main.py

```diff
@@ -222,13 +222,7 @@ def _print_cluster_config(cluster_config: Dict):
         "server_connection_type",
     ]
 
-    backend_config = [
-        "resource_subtype",
-        "domain",
-        "server_host",
-        "ips",
-        "resource_subtype",
-    ]
+    backend_config = ["resource_subtype", "domain", "server_host", "ips"]
```

Comment:
why is resource_subtype removed?

Reply:
Because it was added twice to the `backend_config` list, by mistake. Not it appears only once.